test/e2e/resources/version-configmap.yaml

```diff
@@ -0,0 +1,19 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: pravega-supported-upgrade-paths
+data:
+  0.1.0: "0.1.0"
+  0.2.0: "0.2.0"
+  0.3.0: "0.3.0,0.3.1,0.3.2"
+  0.3.1: "0.3.1,0.3.2"
+  0.3.2: "0.3.2"
+  0.4.0: "0.4.0"
+  0.5.0: "0.5.0,0.5.1,0.6.0,0.6.1,0.6.2,0.6.3,0.7.0,0.8.0"
+  0.5.1: "0.5.1,0.6.0,0.6.1,0.6.2,0.6.3,0.7.0,0.8.0"
+  0.6.0: "0.6.0,0.6.1,0.6.2,0.6.3,0.7.0,0.8.0"
+  0.6.1: "0.6.1,0.6.2,0.6.3,0.7.0,0.8.0"
+  0.6.2: "0.6.2,0.6.3,0.7.0,0.8.0"
+  0.6.3: "0.6.3,0.7.0,0.8.0"
+  0.7.0: "0.7.0,0.8.0"
+  0.8.0: "0.8.0"
```

Comment:
Does this map need to have 0.8.0?? I guess not.

Reply:
We are using the same hardcoded map values in 3-4 different places. Tomorrow when a new release is created we would need to update in all these 3-4 places. Can we do better?

Reply:
The map is now hardcoded in 2 places based on whether the user wants to install the cluster manually or using charts. The map in the test file does not need to be updated.